Dockerfile.sdk

```diff
@@ -168,5 +168,13 @@ ARG TRITON_MODEL_ANALYZER_REPO_TAG
 ARG TRITON_MODEL_ANALYZER_REPO="https://github.com/triton-inference-server/model_analyzer@${TRITON_MODEL_ANALYZER_REPO_TAG}"
 RUN pip3 install "git+${TRITON_MODEL_ANALYZER_REPO}"
 
+# Entrypoint Banner
+ARG NVIDIA_TRITON_INFERENCE_SERVER_SDK_VERSION=21.10
```

Comment:
We don't want to hardwire this verision. Can you instead get it from the TRITON_VERSION file?

Reply:
Will be passed in from CI, changed 21.10 default to `unknown`